src/spikeinterface/core/base.py

```diff
@@ -985,7 +1013,7 @@ def _make_paths_relative(d, relative) -> dict:
 
 def _make_paths_absolute(d, base):
     base = Path(base)
-    func = lambda p: str((base / p).resolve().absolute())
+    func = lambda p: str((base / p).resolve())
```

Comment:
why ?

Reply:
Redundant. Resolve does what absolute does and better:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve